common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -402,24 +405,13 @@ class @Problem
             MathJax.Hub.Queue ["Typeset", MathJax.Hub, element]
 
         `// Translators: the word Answer here refers to the answer to a problem the student must solve.`
-        @$('.show-label').text gettext('Hide Answer')
+
         @el.addClass 'showed'
+        $('html, body').animate({
+          scrollTop: @questionTitle.offset().top
+        }, 500);
         @updateProgress response
         window.SR.readElts(answer_text)
```

Comment:
Mark and I talked in person. He's fine with this functionality staying place (currently only used for capa Show Answer and correct/incorrect status change). @cptvitamin will keep an eye on this in a11y reviews, as the correct/incorrect update may interfere with the Submit functionality.